### PR TITLE
Denis mwangangi

### PR DESCRIPTION
**Pull request by Mwangangi1978**

### CODE REVIEW

File: docs/src/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from "react"
+
+export default function Home() {
+    return (
+        <>
+            
+        <div div class=" bg-white p-4 text-lg font-semibold rounded-2xl selection-box">
+            <h1 className="font-bold text-center bg-black text-white rounded-2xl">Tic-Tac-Toe</h1>
+            <h2 className="font-bold"> Choose the Icon you shall use</h2>
```
Comment (theeurbanlegend) [inline docs/src/Home.jsx:9]: Could we say like choose player??

Comment (itsbluejelly) [reply docs/src/Home.jsx:9]: yeah, i think its okay

Comment (Mwangangi1978) [reply docs/src/Home.jsx:9]: bora game inawork

Comment (theeurbanlegend) [reply docs/src/Home.jsx]: yeah usikuwe na violence im just praco 😆

**Pull request merged**